docs/content/en/docs/architecture/components/scheduler/_index.md

```diff
@@ -5,42 +5,54 @@ description: Basic understanding of how Keptn integrates with Kubernetes Pod Sch
-and orchestrate the deployment process.
+* On Kubernetes versions 1.26 and older,
+  Keptn uses the **Keptn Scheduler** to block application deployment when appropriate
+  and orchestrate the deployment process.
 
-If the Keptn helm chart value `schedulingGatesEnabled` is set to `true`, and Keptn is running on a Kubernetes version
-greater than 1.26, Keptn does not install a scheduler plugin.
-Instead, it uses
-the [Pod Scheduling Readiness K8s API](https://kubernetes.io/docs/concepts/scheduling-eviction/pod-scheduling-readiness)
-to gate Pods until the required deployment checks pass.
+* On Kubernetes version 1.27 and greater,
+  scheduling is implemented using
+  [Kubernetes scheduling gates](https://kubernetes.io/docs/concepts/scheduling-eviction/pod-scheduling-readiness/).
+
+These two implementations are discussed below.
 
 ## Keptn Scheduling Gates for K8s 1.27 and above
 
-When you apply a workload to a K8s cluster,
-the Mutating Webhook checks each Pod for annotations
-to see if it is annotated with
-[Keptn specific annotations](../../../implementing/integrate/#basic-annotations).
-If the annotations are present, the Webhook adds a gate to the Pod called `keptn-prechecks-gate`.
+When Keptn is running on Kubernetes version 1.27 and greater
+and the Keptn Helm chart value `lifecycleOperator.schedulingGatesEnabled` is set to `true`,
+Keptn uses the
+[Pod Scheduling Readiness K8s API](https://kubernetes.io/docs/concepts/scheduling-eviction/pod-scheduling-readiness)
+to gate Pods until the required deployment checks pass.
+
+When a workload is applied to a K8s cluster,
+the Mutating Webhook checks each Pod for annotations.
+If
+[Keptn specific annotations](../../../implementing/integrate/#basic-annotations)
+are present,
+the Webhook adds a gate to the Pod called `keptn-prechecks-gate`.
 This spec tells the Kubernetes scheduling framework
 to wait for  the Keptn checks before assigning the pod to a node.
 
-For instance a pod gated by keptn looks like the following:
+For example, a pod gated by Keptn looks like the following:
 
 {{< embed path="/docs/assets/scheduler-gates/gated.yaml" >}}
 
-The **WorkloadVersion CRD** contains information about the `pre-deployment` checks that
-need to be performed before the Pod can be scheduled.
-If the `pre-deployment` checks have finished successfully, the WorkloadVersion Controller removes the gate from the
-Pod.
-The scheduler can then allow the Pod to be scheduled to a node.
-If the `pre-deployment` checks have not yet finished, the gate stays and the Pod remains in the pending state.
-When removing the gate, the WorkloadVersion controller also adds the following annotation so that,
-if the spec is updated,
-the Pod is not gated again:
+The
+[WorkloadVersion](../../../crd-ref/lifecycle/v1alpha4/#keptnworkloadversion)
+CRD contains information about the `pre-deployment` checks
+that need to be performed before the Pod can be scheduled.
```

Comment:
Users won't create this CR. I suggest removing this sentence entirely since it doesn't introduce any value